src/app/core/relations/relations.js

```diff
@@ -111,32 +181,58 @@ proto.removeRelation = function(relation) {
   }
 };
 
+/**
+ * @param {string} fatherId relation identifier
+ * @returns {boolean}
+ */
 proto.hasChildren = function(childId) {
   const children = this.getChildren(childId);
-  return  children ? !!children.length: false;
+  return children ? !!children.length : false;
 };
 
+/**
+ * @param {string} fatherId relation identifier
+ * @returns {boolean}
+ */
 proto.hasFathers = function(fatherId) {
   const fathers = this.getFathers(fatherId);
   return fathers ? !!fathers.length : false;
 };
 
-// get children based on father id
+/**
+ * Get children based on father id
+ * 
+ * @param {string} fatherId relation identifier
+ * @returns {object} children
+ */
 proto.getChildren = function(fatherId) {
   if (!this.isFather(fatherId)) return null;
   return this._relationsInfo.fathers[fatherId];
 };
 
-// get fathers based on childId
+/**
+ * Get fathers based on childId
+ * 
+ * @prop {string} childId relation identifier
+ * @returns {object} fathers
+ */
 proto.getFathers = function(childId) {
   if (!this.isChild(childId)) return null;
   return this._relationsInfo.children[childId];
```

Comment:
Here the code is not very clear (there seems to be a mistake), why calling the following function:

```js
Relations::getFathers(childId)
```

is the children returned?

```js
return this._relationsInfo.children[childId];
```

Reply:
@Raruto  this._relationsInfo.children store layers that are child in relations and store layerId fathers